advanced-vector-search/tests/test_advanced_vectors_search.py

```diff
@@ -8,52 +8,20 @@
 
 DATASET_NAME = 'siftsmall'
 
-
-@pytest.fixture(scope='session')
-def configuration():
-    os.environ['JINA_DATASET_NAME'] = DATASET_NAME
-    os.environ['JINA_USES_FAISS'] = 'docker://faiss_indexer_image:test'
-    os.environ['JINA_USES_ANNOY'] = 'docker://annoy_indexer_image:test'
-    general_config()
-    yield
-    del os.environ['JINA_DATASET_NAME']
-    del os.environ['JINA_USES_FAISS']
-    del os.environ['JINA_USES_ANNOY']
-
-
 @pytest.fixture(scope='session')
-def sift_data(configuration):
+def sift_data():
```

Comment:
I think `configuration` is needed. Is not about dependency as parameter, but it makes sure that is processed before

Reply:
Seems like it's passing without. Are you sure it's needed?